intro-HPC/ch_running_jobs_with_input_output_data.tex

```diff
@@ -231,7 +231,11 @@ \subsection{Your data directory (\$VSC\_DATA)\label{subsec:data-directory}}
 \ifgent
 If you are running out of quota on your \$VSC\_DATA filesystem you can
 request a VO (Virtual organisation). VO members get an extra DATA filesystem: \$VSC\_DATA\_VO)
-We can give higher quota values for these filesystems. Don't hesitate to contact the \hpcTeam staff.
+We can give higher quota values for these filesystems. VO administrators can request additional quota
+at the "Edit VO" page \url{https://account.vscentrum.be/django/vo/edit}.
+Do note that additional quota is an upper bound that can be raised, but this is not
+a guarantee you will be able to use the additional quota since we do oversubscribe our storage
+to achieve optimal usage.
```

Comment:
It could happen, and we want to make this clear, better inform users about this so they are not under the false impression that everyone can use all the quota they receive.